types/sql-template/sql-template-tests.ts

```diff
@@ -0,0 +1,21 @@
+import * as SQL from "sql-template";
```

Comment:
Since this is a callable or constructable entity, it isn't legal in ES2015 to import and call/construct this with a namespace import like `import * as foo from 'foo'`, so could you please convert this to either

```ts
import foo = require('foo')
```

or the following with `esModuleInterop` turned on

```ts
import foo from 'foo';
```

Reply:
I've corrected the  import in the test script.